src/elements/content-sidebar/activity-feed/task-new/__tests__/__snapshots__/Assignees-test.js.snap

```diff
@@ -0,0 +1,921 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`elements/content-sidebar/ActivityFeed/task-new/Assignees should open assignee list when overflow icon is clicked: assignment list 1`] = `
```

Comment:
These are pretty humungous. Would using `shallow` provide a clearer view of the rendered component hierarchy?

Reply:
I don't know if they are super valuable really - it's a bunch of Tooltips and Overlay classes. Will probably delete in the next pass.